pkg/devices/power.go

```diff
@@ -0,0 +1,96 @@
+package devices
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/vapor-ware/synse-emulator-plugin/pkg/utils"
+	"github.com/vapor-ware/synse-sdk/sdk"
+)
+
+// Power is the handler for the emulated power device(s).
+var Power = sdk.DeviceHandler{
+	Name:  "power",
+	Read:  powerRead,
+	Write: powerWrite,
+}
+
+// powerRead is the read handler for the emulated power device(s).
+// It returns random values between 0 and 100.
+func powerRead(device *sdk.Device) ([]*sdk.Reading, error) {
+	// Default reading ranges
+	var min = 0
+	var max = 100
```

Comment:
There are some negative values... I have no idea why. The expected range IMO, would be between 1000-3000 for the `Total` values. To reduced needless complexity, we can use that same range for the individual Zone power levels rather than try to simulate an aggregate. 
![image](https://user-images.githubusercontent.com/9396966/59770367-87444680-926d-11e9-849b-be62a97b5d8d.png)